e2e/cypress/integration/toast/permalink_jump_to_spec.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// # Indicates a test step (e.g. # Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @toast
+
+describe('Permalink', () => {
+    let testTeam;
+
+    before(() => {
+        // # Create new team and new user
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            testTeam = team;
+        });
+    });
+
+    it('MM-T1791 Permalink \'Jump to\' in Search', () => {
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Search for a term e.g.test
+        const searchTerm = 'test';
+        cy.get('#post_textbox').type(searchTerm).type('{enter}');
```

Comment:
Please use `cy.postMessage` to take advantage of internal wait to textbox.
```suggestion
        cy.postMessage(searchTerm);
```